### ISSUE TITLE
a11y P3 - Updated timestamp requires 2 swipes on iOS voice over

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#5515]
**Describe the bug**
The updated time stamp takes 2 swipes to read out, it should be just one. Looks like there's a comment separating the translated text and the date. The comment might be being added by React, doesn't look like we are explicitly doing it https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/index.jsx#L56-L59

**To Reproduce**
Steps to reproduce the behaviour:

1. Enable Voice Over on iOS on iPad/iPhone
2. Navigate to PGL page

**Expected behaviour**
The updated timestamp should only require one swipe to read out content.

**Screenshots**
<img width="234" alt="Screen Shot 2020-02-11 at 15 44 46" src="https://user-images.githubusercontent.com/4706195/74328878-9e98ec80-4d86-11ea-9ec4-3a4b2b7b2585.png">

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by sareh** [comment]
This issue can be fixed by creating a constant further up that does the concatenation and then rendering that contant within the Timstamp component `<Timestamp ...>{timestampText}</Timestamp>`
